nautobot/utilities/testing/api.py

```diff
@@ -233,6 +234,27 @@ def test_options_objects_returns_display_and_value(self):
                     self.assertIn("display", choice, f"A choice in {field} is missing the display key")
                     self.assertIn("value", choice, f"A choice in {field} is missing the value key")
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
+        def test_options_returns_expected_choices(self):
+            """
+            Make an OPTIONS request for a list endpoint and validate choices match expected choices.
+            """
+            # Skip test if test case does not supply a list of choices to test against.
+            if not self.choices:
+                self.skipTest("choices was not provided by test case.")
```

Comment:
Would it instead make sense to validate that *no* OPTIONS choices are returned in this case? I recognize that this might require adding `choices` declarations to a bunch of existing test cases for correctness but that would probably be a good thing to do regardless.